src/main/java/uk/nhs/hee/tis/revalidation/connection/repository/CurrentConnectionElasticSearchRepository.java

```diff
@@ -27,15 +27,14 @@
 import org.springframework.data.elasticsearch.repository.ElasticsearchRepository;
 import org.springframework.stereotype.Repository;
 import uk.nhs.hee.tis.revalidation.connection.entity.CurrentConnectionsView;
-import uk.nhs.hee.tis.revalidation.connection.entity.ExceptionView;
 
 @Repository
 public interface CurrentConnectionElasticSearchRepository
     extends ElasticsearchRepository<CurrentConnectionsView, String> {
 
-  @Query("{\"bool\":{\"filter\":[{\"match\":{\"designatedBody\":\"?1\"}},"
-      + "{\"bool\":{\"should\":"
-      + "[{\"wildcard\":{\"doctorFirstName\":{\"value\":\"?0*\"}}},"
+  @Query("{\"bool\":{\"must_not\":{\"match\":{\"membershipType\":\"MILITARY\"}},"
```

Comment:
I can see someone saying "Oh wait, except for when those Military trainees..." & want to make sure that the way those trainees are revalidated is clear.  A topic for discussion at review, perhaps!